packages/react-component-library/package.json

```diff
@@ -48,6 +48,7 @@
     "@types/node": "^12.0.2",
     "@types/react": "^16.8.18",
     "@types/react-dom": "^16.8.4",
+    "@types/react-test-renderer": "^16.8.1",
```

Comment:
what about enzyme?

Reply:
Changed to enzyme-to-json for snapshot serialisation.